apps/reference/docs/guides/realtime/limits.mdx

```diff
@@ -0,0 +1,66 @@
+---
+id: limits
+title: Realtime Limits
+description: Realtime Rate Limiting
+---
+
+Since deploying a global multi-tenant cluster for Supabase Realtime, we've implimented some rate limits to protect our infrastructure.
+
+Rate limits are mostly configurable per project and our cluster can support millions of concurrent connections if needed. If you're running into any rate limiting and it's causing issues, please [contact support](https://app.supabase.com/support/new).
+
+## Limits by Plan
+
+Limits are set on the Free and Pro plans accordingly. See our [pricing page](https://supabase.com/pricing) for more information.
+
+Above the Pro plan Realtime is billed on a usage basis. We still employ limiting above the Pro plan to protect our infrastructure. If you're on an Enterprise plan simply contact support and we'll increase your limits as needed.
+
+Enterprise plan limits start at:
+
+- 500 concurrent users
+- 1,000 messages per second
+- 500 concurrent Channels
+
+## System Limits
+
+Some limits apply to all projects and are not currently configurable per project.
+
+- 500 Channel joins per second
+- 100 Channels per connected user
+
+## Client-side Limiting
+
+Some basic WebSocket message rate limiting has been implimented client-side.
+
+See the [Cursor Positions Quickstart](/docs/guides/realtime/quickstart#cursor-positions) section where a `eventsPerSecond` parameter is passed in when instantiating the Supabase client.
+
+- Default: `10` events per second
+
+## Rate Limiting Errors
+
+Errors could be generated in two places: the backend logs, a message in WebSocket.
+
+:::note
+Use the [Realtime Inspector](https://realtime.supabase.com/inspector/new) to reproduce an error and share those connection details with Supabase support.
```

Comment:
We should let them know that when sharing the token comes in the url so be careful not to expose tokens like `service_role`.

Reply:
The Inspector says to use the anon key, but I think I'm going to pull that out of the share url anyways.